modules/release-notes/pages/index.adoc

```diff
@@ -122,6 +122,28 @@ vertex function can also be used with vertex input parameters in the query-body
 
 == Fixed issues
 
+=== Fixed and Improved in 3.9.2-1
+
+==== Crashes and Deadlocks
+* Fixed a GPE crash caused by aborted query still being used by running thread.
+* Fixed crash issue after RESTPP auth refresh.
+* Fixed GSE crash due to the check failure when refreshing leader info after 60 minute timeout.
+* Fixed GPE crash and non-recovery due to Kafka retention to a higher offset.
+
+==== Functionality
+* Fixed data inconsistency while loading data and post-processing operations
```

Comment:
Can we investigate the scenario in hopes of narrowing down "post-processing" to a more limited scenario?

Reply:
Asked a question to the ticket assignee on slack and added you for visibility. Hoping to get an answer by tomorrow.

Reply:
I made some changes to this and is reflected in the changes based on the conversation with the engineer. Resolving this conversation but there is a note on new ones.